pyQuARC/code/gcmd_validator.py

```diff
@@ -2,7 +2,7 @@
 import os
 import requests
 
-from pyQuARC.code.utils import get_headers
+from .utils import get_headers
```

Comment:
This might work when not using package, but does it work when using "installed" package?

Reply:
@svbagwell and I are looking into this. To be sure, what is meant by "'installed' package"?

Reply:
pyQuARC is a package. once you install it using `python setup.py install`, does it work the same in any python shell run outside of pyQuARC folder?

Reply:
this seems to be working for me, i did not get any errors